wgpu/Cargo.toml

```diff
@@ -130,6 +130,12 @@ features = ["raw-window-handle"]
 workspace = true
 features = ["raw-window-handle"]
 
+# If we are not targeting WebGL, enable indirect-validation.
```

Comment:
nitpick: I think this covers more than just the WebGL target. I think this covers WASM _and_ Emscripten targets for WebGPU _and_ WebGL backends? Seems like something we should clarify?

Reply:
That is true but `wgpu-core` is not used when `wgpu` targets WebGPU.

Reply:
🤷🏻‍♂️